tests/datatypes/test_list.py

```diff
@@ -184,3 +176,88 @@ class InplaceListOperationTests(InplaceOperationTestCase, TranspileTestCase):
 
         'test_modulo_complex',
     ]
+
+class ListComparisonTests(TranspileTestCase):
```

Comment:
By convention, these tests are at the top of the test file, not the bottom; see [test_dict.py](https://github.com/pybee/batavia/blob/master/tests/datatypes/test_dict.py) for an example